### ISSUE TITLE
utf-8

### ISSUE THREAD

**Issue by amenks** [oziresrds/html-to-rtf#20]
I am facing encoding problems.
With this code:

`var htmlToRtf = require('html-to-rtf');`
`var html = "<p>comparação</p>"`
`htmlToRtf.saveRtfInFile('try.rtf', htmlToRtf.convertHtmlToRtf(html))`

my rtf file comes with:

comparaÃ§Ã£o 

Am I doing something wrong?

**Comment by amenks** [comment]
I found a way.
After a little research about the ansi encoding I found  the ansi-encode module.
With the code in this way:

`const htmlToRtf = require('html-to-rtf');`
`const ansiEncode = require("ansi-encode");`
`const  html = "<p>comparação</p>"`
`const ansiHtml = ansiEncode(html).replace(/u00/gi,"'")`
`htmlToRtf.saveRtfInFile('try.rtf', htmlToRtf.convertHtmlToRtf(ansiHtml))`

It is not pretty but works.
